src/core/searchUtils.js

```diff
@@ -35,6 +41,14 @@ export function convertQueryParamsToFilters(params) {
   }, {});
 }
 
+export function convertOperatingSystemToFilterName(name) {
+  if (name in operatingSystems) {
+    return operatingSystems[name];
+  }
+
```

Comment:
It would be nice to see a warning in the log about unknown operating systems. I suspect we'll have to fix some edge cases like `Linux (x86)` due to UA parsing errors -- just a hunch! Hopefully not.

Reply:
That's totally true, thanks.